configstack/running_module.go

```diff
@@ -236,7 +264,9 @@ func (this MultiError) ExitStatus() (int, error) {
 	for i := range this.Errors {
 		if code, err := shell.GetExitCode(this.Errors[i]); err != nil {
 			return -1, this
-		} else if code > exitCode {
+		} else if code == 1 || code == 2 && exitCode == 0 {
+			// The exit code 1 is more significant that the exit code 2 because it represents an error
+			// while 2 represent a warning.
```

Comment:
The modules themselves would have to change their error codes before returning them